test/e2e/performanceprofile/functests/6_mustgather_testing/mustgather.go

```diff
@@ -39,60 +39,68 @@ var _ = Describe("[rfe_id: 50649] Performance Addon Operator Must Gather", func(
-		It("[test_id:50650][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Verify Generic cluster resource definitions are captured", func() {
+		It("Verify Generic cluster resource definitions are captured", func() {
 			var genericFiles = []string{
 				"version",
 				"cluster-scoped-resources/config.openshift.io/featuregates/cluster.yaml",
 				"namespaces/openshift-cluster-node-tuning-operator/tuned.openshift.io/tuneds/default.yaml",
 				"namespaces/openshift-cluster-node-tuning-operator/tuned.openshift.io/tuneds/rendered.yaml",
 			}
-			err := checkfilesExist(genericFiles)
+			err := checkfilesExist(genericFiles, mustGatherContentDir)
 			Expect(err).To(BeNil())
 		})
 
-		It("[test_id:50651][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Verify PAO cluster resources are captured", func() {
+		It("Verify PAO cluster resources are captured", func() {
 			profile, _ = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
 			if profile != nil {
+				//replace peformance.yaml for profile.Name when data is generated in the node
 				ClusterSpecificFiles := []string{
-					fmt.Sprintf("cluster-scoped-resources/performance.openshift.io/performanceprofiles/performance.yaml"),
-					fmt.Sprintf("cluster-scoped-resources/machineconfiguration.openshift.io/kubeletconfigs/performance-%s.yaml", profile.Name),
-					fmt.Sprintf("namespaces/openshift-cluster-node-tuning-operator/tuned.openshift.io/tuneds/openshift-node-performance-%s.yaml", profile.Name),
+					"cluster-scoped-resources/performance.openshift.io/performanceprofiles/performance.yaml",
+					"cluster-scoped-resources/machineconfiguration.openshift.io/kubeletconfigs/performance-manual.yaml",
+					"namespaces/openshift-cluster-node-tuning-operator/tuned.openshift.io/tuneds/openshift-node-performance-manual.yaml",
 				}
-				err := checkfilesExist(ClusterSpecificFiles)
+				err := checkfilesExist(ClusterSpecificFiles, mustGatherContentDir)
 				Expect(err).To(BeNil())
 			} else {
 				Skip("No Performance Profile found")
 			}
 		})
-		It("[test_id:50652][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Verify hardware related information are captured", func() {
-			workerRTNodes, err = nodes.GetByLabels(testutils.NodeSelectorLabels)
-			Expect(err).ToNot(HaveOccurred())
-			workerRTNodes, err = nodes.MatchingOptionalSelector(workerRTNodes)
-			Expect(err).ToNot(HaveOccurred())
-			for _, node := range workerRTNodes {
-				nodeSpecificFiles := []string{
-					fmt.Sprintf("nodes/%s/%s_logs_kubelet.gz", node.Name, node.Name),
-					fmt.Sprintf("nodes/%s/lscpu", node.Name),
-					fmt.Sprintf("nodes/%s/lspci", node.Name),
-					fmt.Sprintf("nodes/%s/proc_cmdline", node.Name),
-				}
-				err := checkfilesExist(nodeSpecificFiles)
-				Expect(err).To(BeNil())
+		It("Verify hardware related information are captured", func() {
+			nodeSpecificFiles := []string{
+				"/lscpu",
+				"/lspci",
+				"/proc_cmdline",
+				"/sys/devices/system/cpu/offline",
```

Comment:
this also is expected to be a relative path, the heading slash is misleading

Reply:
done